src/evo/netinfo.cpp

```diff
@@ -5,6 +5,7 @@
 #include <evo/netinfo.h>
 
 #include <chainparams.h>
+#include <evo/providertx.h>
```

Comment:
let's create better new header `evo/protx_version.h` and move there this enum.

Or, alternativaly, rename header `evo/dmn_types.h` to evo/types.h or evo/versions.h and have there both ProTxVersion and MnType.